### ISSUE TITLE
Not working with UNC paths

### ISSUE THREAD

**Issue by WebDucer** [cake-contrib/Cake.LongPath.Module#14]
Hello.

If I use the module `Cake.LongPath.Module` in my cake scripts, I am not able to use UNC paths. For example:

```csharp
CleanDirectory(deployDirectory);
// OR
CopyFiles(deployFiles, deployDirectory);
```

produce an exception

```
An error occurred when executing task '_deployWeb'.
Error: System.AggregateException: Mindestens ein Fehler ist aufgetreten. ---> System.IO.IOException: Der Vorgang wurde erfolgreich beendet.

   bei Pri.LongPath.Common.ThrowIOError(Int32 errorCode, String maybeFullPath)
   bei Pri.LongPath.FileSystemInfo.Refresh()
   bei Pri.LongPath.DirectoryInfo.get_Exists()
   bei Cake.Common.IO.DirectoryCleaner.Clean(ICakeContext context, DirectoryPath path, Func`2 predicate)
   bei Cake.Common.IO.DirectoryAliases.CleanDirectory(ICakeContext context, DirectoryPath path)
   bei Submission#0.CleanDirectory(DirectoryPath path)
   bei Submission#0.<<Initialize>>b__0_14()
   bei Cake.Core.CakeTaskBuilderExtensions.<>c__DisplayClass20_0.<Does>b__0(ICakeContext x)
   bei Cake.Core.CakeTask.<Execute>d__43.MoveNext()
```

Without the module, the same works fine.

**Comment by devlead** [comment]
Which version of Cake are you using? Having same issue with just released 0.31.0?

Could be an error in upstream [Pri.LongPath](https://github.com/peteraritchie/LongPath) assembly used, unsure if it supports UNC.

**Comment by WebDucer** [comment]
@devlead Thanks. With the new Cake version (0.31.0) it works as expected.
